### PR TITLE
Release v1.13.1 [sc-1076]

### PR DESCRIPTION
**Pull request by teghnet**
Supersedes: #28

### CODE REVIEW

File: omnia/config/relay.conf [hunk 1]
```diff
@@ -122,6 +122,12 @@
       "oracle": "0x89AC26C0aFCB28EC55B6CD2F6b7DAD867Fa24639",
       "oracleExpiration": 86400,
       "oracleSpread": 4
+    },
+    "RETH/USD": {
+      "msgExpiration": 1800,
+      "oracle": "0xf86360f0127f8a441cfca332c75992d1c692b3d1",
```
Comment (MDobak) [inline omnia/config/relay.conf:128]: I noticed, that the `wat` parameter in that contract is set to `ethusd` instead of `rethusd`.

Comment (NiklasKunkel) [reply omnia/config/relay.conf:128]: we need to deploy a new Medianizer and update the config, tag a new omnia release, and include it in the oracles 1.13.1 release

Comment (jar-o) [reply omnia/config/relay.conf:128]: @MDobak that is just the default value on the `Median` contract, which is overridden by the `MedianRETHUSD` contract, see line 182

https://etherscan.io/address/0xf86360f0127f8a441cfca332c75992d1c692b3d1#code

**Comment by NiklasKunkel** [comment]
lgtm

**Pull request merged**